src/index.js

```diff
@@ -0,0 +1,154 @@
+/* 기능 구현 1. 사용자 입력 처리 & 잘못된 값 입력 판별 기능
+  - 규칙 : 길이 3, 각 자리 1~9 숫자, 중복 안됨
+  - 잘못된 입력이면 alert 띄우기
+*/
+
+export default function BaseballGame() {
+  this.play = function (computerInputNumbers, userInputNumbers) {
+    const computerInput = String(computerInputNumbers).trim();
+    const userInput = String(userInputNumbers).trim();
+
+    if (!isValidInput(userInput)) {
+      window.alert("올바르지 않은 입력입니다.");
+      return "";
+    }
+    const { ball, strike } = judgeCorrect(computerInput, userInput);
+    return resultJudge(ball, strike);
+  };
+}
+
+// 유저가 입력한 값이 유효한지 검사하는 함수
+function isValidInput(userInput) {
+  // 1) 길이가 3이 아니면 false
+  if (userInput.length !== 3) return false;
+
+  // 2) 각 자리 숫자가 1~9인지 검사
+  const first = userInput[0],
+    second = userInput[1],
+    third = userInput[2];
```

Comment:
[구조 분해 할당](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring)을 이용한다면 이렇게 선언해서 각 입력값의 자릿수를 저장할 수도 있을 것 같은데 어떻게 생각하시나요?
```suggestion
  const [first, second, third] = userInput;
```

Reply:
구조 분해 할당 자료를 읽고 정리도 해놔서 알고 있다고 생각했는데, 막상 사용할 수 있는 부분을 놓치고 있었다는 것을 깨달았습니다. 훨씬 깔끔한 방법이라고 생각합니다! 
코드에서는 숫자야구 게임의 자릿수를 변경할 수 있도록 수정해서 반영하지 못했지만, 앞으로 구조분해 할당이 사용될 수 있는 부분은 적용해보겠습니다!

Reply:
코드 확인했어요, 앞으로의 변경에 강하겠는걸요?